src/Interpreters/Aggregator.cpp

```diff
@@ -1435,13 +1435,14 @@ void NO_INLINE Aggregator::mergeOnIntervalWithoutKey(
     AggregatedDataVariants & data_variants,
     size_t row_begin,
     size_t row_end,
-    const AggregateColumnsConstData & aggregate_columns_data) const
+    const AggregateColumnsConstData & aggregate_columns_data,
+    std::atomic<bool> * is_cancelled) const
```

Comment:
Why pointer instead of reference in all places if we never pass `nullptr`?

Reply:
Updated.